content/blueprints/scaling.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: bt_wiki
+title: Scaling
+category: Blueprints
+draft: false
+weight: 1600
+
+---
+
+This page describes different concepts involved in scaling components of your application blueprint.
+
+## Overview
+When we refer to scaling, we are actually referring to the number of node instances each node template will have on deployment, and changes made
+to that number during runtime after the deployment is created, using the `scale` workflow.
+
+There are two ways of specifying this configuration in the application blueprint. We will show both and what will follow is an explanation of
+the difference between these two methods.
+
```

Comment:
maybe ... between these two methods, after which we will show how the number of instances...